programs/dynamic-bonding-curve/src/instructions/migration/dynamic_amm_v2/migrate_damm_v2_initialize_pool.rs

```diff
@@ -191,20 +191,7 @@ impl<'info> MigrateDammV2Ctx<'info> {
     ) -> Result<()> {
         let pool_authority_seeds = pool_authority_seeds!(bump);
 
-        // Send some lamport to pool authority to pay rent fee?
-        msg!("transfer lamport to pool_authority");
-        invoke(
-            &system_instruction::transfer(
-                &self.payer.key(),
-                &self.pool_authority.key(),
-                50_000_000, // TODO calculate correct lamport here
-            ),
-            &[
-                self.payer.to_account_info(),
-                self.pool_authority.to_account_info(),
-                self.system_program.to_account_info(),
-            ],
-        )?;
+        let before_lamports = self.pool_authority.lamports();
```

Comment:
could you comment, in order to do flash rent, someone need to fund `pool_authority ` at least 1 SOL when program is newly created (in case someone fork the code and run by themselves)

Reply:
Fixed